meson.build

```diff
@@ -0,0 +1,46 @@
+#
+# See LICENSE for more information about licensing
+#  Copyright 2023
+#
+# Author: Diego Arturo Avila Torres <dandida95@gmail.com>
+#         Luis G. Leon Vega <luis.leon@ieee.org>
+#
+#
+
+project(
+  'cynq',
+  'cpp',
+  version : '0.1.0',
```

Comment:
Put this into a single line. Too many lines for this instruction. Save scrolling